server/server.go

```diff
@@ -1270,6 +1270,9 @@ func (s *Server) StartProfiler() {
 	s.profilingServer = srv
 	s.mu.Unlock()
 
+	// Enable blocking profile
+	runtime.SetBlockProfileRate(1)
```

Comment:
I am thinking this was part of a test, no?

Reply:
It has to be set, so if you wanted to profile without it you have to change source code. WDYT?

Reply:
Right, so yes, leave it.